vl53l1x/vl53l1x.go

```diff
@@ -334,51 +331,44 @@ func (d *Device) getRangingData() {
 	d.rangingData.mm = uint16((uint32(d.results.mmCrosstalkSD0)*2011 + 0x0400) / 0x0800)
 	switch d.results.status {
 	case 17: // MULTCLIPFAIL
```

Comment:
These lines could be reduced to

`case 17, 2, 1, 3`

instead of using `fallthrough`

Reply:
I've been discussing exactly this with my colleague, but we opted for keeping the comments and going with 'fallthrough'.
How about this _middle ground_?

Reply:
Seems good to me. Thanks @ostaebler